### PR TITLE
Feature/cookie consent

### PR DESCRIPTION
**Pull request by MartinGerritsen**
https://trello.com/c/aI6OyPpd/101-a%C3%B1adir-un-banner-de-aceptaci%C3%B3n-de-cookies

image

Updated Modal to work with a specific ID to have more than one in page
Added CookieConsent component with a generic text
On denied redirect to google.com
On accept saved in localstorage
Redirect link to politica-privacidad

### CODE REVIEW

File: src/components/CookieBanner/CookieBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+import { FC, useEffect, useState } from 'react';
+import Modal from '@/components/Modal';
+import { useModal } from '@/context/EmergencyProvider';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+
+const MODAL_NAME = 'cookie-banner';
+const POLICY_URL = '/politica-privacidad';
+const COOKIE_CONSENT_KEY = 'ajudadanaCookieConsentAccepted';
```
Comment (patrickwebsdev) [inline src/components/CookieBanner/CookieBanner.tsx:11]: No repetir código, reutiliza la const en constants.ts

File: src/helpers/constants.ts [hunk 2]
```diff
@@ -20,3 +20,5 @@ export const mapToIdAndLabel = (data: any) => {
     label: data[key],
   }));
 };
+
+export const COOKIE_CONSENT_KEY = 'ajudana_cookie_consent';
```
Comment (patrickwebsdev) [inline src/helpers/constants.ts:24]: renombrar `ajudana` a `ajudadana`

**Review by patrickwebsdev — CHANGES_REQUESTED** [review]
Haz los cambios que te deje en comentarios y se va a production

File: src/components/CookieBanner/CookieBanner.tsx (hunk 1 above)
Comment (MartinGerritsen) [reply src/components/CookieBanner/CookieBanner.tsx]: El fallo era no borrarlo en constants.ts, solo lo utilizamos aqui.

File: src/helpers/constants.ts (hunk 2 above)
Comment (MartinGerritsen) [reply src/helpers/constants.ts]: Lo he borrado aqui

**Pull request merged**